lib/orbit_web/router.ex

```diff
@@ -14,13 +14,27 @@ defmodule OrbitWeb.Router do
     plug :accepts, ["json"]
   end
 
+  pipeline :authenticated do
```

Comment:
At the bottom of this file, there's a route for the live dashboard that should also be protected by auth.

Reply:
👍 2bec206